.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["AllenCell/10"]
+assignees:
+  - hughes036
```

Comment:
Intending to hardcode your GH on here?

Reply:
yeah, ill change it later i think.  just testing functionality (which needs to be on main)